appengine/standard/ndb/schema_update/add_entities_handler.py

```diff
@@ -0,0 +1,27 @@
+import os
+import time
+
+import jinja2
+import webapp2
+from google.appengine.ext import ndb
+
+import models_v1
+
+JINJA_ENVIRONMENT = jinja2.Environment(
+    loader=jinja2.FileSystemLoader(os.path.dirname(__file__)),
+    extensions=['jinja2.ext.autoescape'],
+    autoescape=True)
+
+
+class AddEntitiesHandler(webapp2.RequestHandler):
+    def post(self):
+        reload(models_v1)
```

Comment:
looks like i dont need this line anymore because this module doesn't use models_v2